src/backend/defs/rest-api/swagger-root.json

```diff
@@ -19,7 +19,9 @@
     "/discussions": {"$ref": "paths/discussions.json#/discussions"},
     "/discussions/{discussion_id}": {"$ref": "paths/discussions.json#/discussions_{discussion_id}"},
     "/discussions/{discussion_id}/messages": {"$ref": "paths/discussions.json#/discussions_{discussion_id}_messages"},
-    "/raws/{raw_msg_id}": {"$ref": "paths/raws.json#/raws_{raw_msg_id}"}
+    "/raws/{raw_msg_id}": {"$ref": "paths/raws.json#/raws_{raw_msg_id}"},
+    "/tags": {"$ref": "paths/tags.json#/tags"},
+    "/tags/{name}": {"$ref": "paths/tags.json#/tags_{name}"}
```

Comment:
in near futur I will need to list tags related to entites, there is two (or three) options:

- the entity gives a collection of tags
- a sub-route of an entity gives the collection
- each tags gives a list of associated entities